src/main/java/org/lobid/resources/run/AlmaMarcXmlFix2lobidJsonEs.java

```diff
@@ -128,9 +128,9 @@ public void run() {
                 fixVariables.put("rvk.tsv", "../../../../../../lookup-tables/data/rvk.tsv");
                 fixVariables.put("lobidOrganisationsMapping.tsv", "./maps/lobidOrganisationsMapping.tsv");
                 fixVariables.put("hbzowner2sigel.tsv", "./maps/hbzowner2sigel.tsv");
-                fixVariables.put("rpb2.ttl", "../../../../../../vocabs/rpb/rpb2.ttl");
-                fixVariables.put("rpb-spatial.ttl", "../../../../../../vocabs/rpb/rpb-spatial.ttl");
-                fixVariables.put("rpb.ttl", "../../../../../../vocabs/rpb/rpb.ttl");
+                fixVariables.put("lbz-notationen.ttl", "../../../../../../lbz-vocabs/rpb/lbz-notationen.ttl");
+                fixVariables.put("rpb-spatial.ttl", "../../../../../../lbz-vocabs/rpb/rpb-spatial.ttl");
+                fixVariables.put("rpb.ttl", "../../../../../../lbz-vocabs/rpb/rpb.ttl");
```

Comment:
There is no `rpb` directory in https://github.com/hbz/lbz-vocabs . I guess that `rpb`-subdirectory has just to be removed here?

Reply:
you are right!!! thanks!

Reply:
Done!